### ISSUE TITLE
ext/SDBM_File: status in p7 branch

### ISSUE THREAD

**Issue by jkeenan** [atoomic/perl#78]
In this ticket we will track our core-p7 progress on tests in directory `ext/SDBM_File`.

As 2020-07-09 10:20 EDT, we are getting this output from `make test` for the files in this directory.

```
../ext/SDBM_File/t/constants.t ..................................... ok
Unquoted string "pagefh" may clash with future reserved word at t/corrupt.t line 19.
Name "main::pagefh" used only once: possible typo at t/corrupt.t line 19.
../ext/SDBM_File/t/corrupt.t ....................................... ok
../ext/SDBM_File/t/prep.t .......................................... ok
../ext/SDBM_File/t/sdbm.t .......................................... ok
```
These results were generated on an unthreaded build on linux.  The results might differ significantly if run on a threaded build.  There have been subsequent commits to this branch which may have impacted these results.  YMMV.

**Comment by jkeenan** [comment]
Upon further study I believe these warnings are calling attention to a typographical error in blead (and several older versions of perl) which I am flagging in https://github.com/Perl/perl5/issues/17970.

I'm going to proceed to correct the typo here, which will make the file run clean and will enable us to close this ticket.

**Comment by jkeenan** [comment]
Fixed in ef35f3acbbd7562806f978f28c9fa2495da501e4.  All test files in this directory now run clean.  Closing.
